p2p/libp2p/netMessenger.go

```diff
@@ -97,6 +98,7 @@ type ArgsNetworkMessenger struct {
 	P2pPrivateKey         commonCrypto.PrivateKey
 	P2pSingleSigner       commonCrypto.SingleSigner
 	P2pKeyGenerator       commonCrypto.KeyGenerator
+	NetworkType           p2p.NetworkType
```

Comment:
I think we should add the network on the 2 prints from this file:
```
network connection status ... 
network connection metrics ...
```

Reply:
already done, these logs are now part of connectionsHandler: https://github.com/multiversx/mx-chain-communication-go/pull/33/files#diff-d3c10daf4322b3b85154be491f3b8264cd5ad0d9b555b46f096e50b2c199dc1aR428

Reply:
oh, missed that